src/main/java/ladder/domain/Ladder.java

```diff
@@ -2,31 +2,31 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.stream.IntStream;
 
 public class Ladder {
     private final List<Line> lines;
 
-    public Ladder() {
-        this(new ArrayList<>());
-    }
-
     public Ladder(List<Line> lines) {
         this.lines = lines;
     }
 
-    public void add(Line line) {
-        lines.add(line);
-    }
-
-    public int arrivalPoint(int startLine, int startPoint) {
+    public int arrivalPoint(int startPoint) {
         int currentPoint = startPoint;
-        for (int i = startLine; i < lines.size(); i++) {
-            currentPoint = lines.get(i).move(currentPoint);
+        for (Line line : lines) {
+            currentPoint = line.move(currentPoint);
         }
         return currentPoint;
     }
 
     public List<Line> getLines() {
         return lines;
     }
+
+    public static Ladder make(int pointCount, int lineCount) {
+        List<Line> ladderLine = new ArrayList<>();
+        IntStream.range(0, lineCount)
+                .forEach(line -> ladderLine.add(Line.init(pointCount)));
+        return new Ladder(ladderLine);
```

Comment:
```suggestion
        return new Ladder(
                Stream.generate(() -> Line.init(pointCount))
                        .limit(lineCount)
                        .collect(Collectors.toList())
        );
```
스트림을 활용하면 이렇게 쓸 수도 있어요. 😄